openedx/core/djangoapps/verified_track_content/partition_scheme.py

```diff
@@ -88,6 +90,8 @@ def get_group_for_user(cls, course_key, user, user_partition, **kwargs):  # pyli
                 mode_slug,
                 modes=CourseMode.modes_for_course(course_key, include_expired=True, only_selectable=False),
             )
+            if course_mode and CourseMode.is_credit_mode(course_mode):
```

Comment:
Note to reviewers: I don't want to just change only_selectable to True, because I need to map any users in Credit to the "Verified" track for the course.